### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/strings/functions/utf8-encode.xml [hunk 1]
```diff
@@ -79,6 +83,12 @@
       </row>
      </thead>
      <tbody>
+      <row>
+       <entry>8.2.0</entry>
+       <entry>
+        Cette fonction a été rendu obsolète.
```
Comment (devnexen) [inline reference/strings/functions/utf8-encode.xml:89]: rendu => rendue but I see the issue in other places too.

Comment (saundefined) [reply reference/strings/functions/utf8-encode.xml]: Thanks for the review! Fixed =)

File: reference/datetime/datetimeinterface/format.xml [hunk 2]
```diff
@@ -174,11 +174,36 @@
             cette année sera utilisée à la place.</entry>
            <entry>Exemples : <literal>1999</literal> ou <literal>2003</literal></entry>
           </row>
+          <row>
+           <entry><literal>X</literal></entry>
+           <entry>
+            Une représentation numérique complète étendue d'une année, au moins 4 chiffres,
```
Comment (adoy) [inline reference/datetime/datetimeinterface/format.xml:180]:
```suggestion
            Une représentation numérique complète étendue d'une année, d'au moins 4 chiffres,
```

File: reference/datetime/datetimeinterface/format.xml [hunk 3]
```diff
@@ -174,11 +174,36 @@
...
+            avec <literal>-</literal> pour les années avant de l'ère commune
```
Comment (adoy) [inline reference/datetime/datetimeinterface/format.xml:181]:
```suggestion
            avec <literal>-</literal> pour les années avant l'ère commune
```

**Review by adoy — CHANGES_REQUESTED** [review]

**Comment by devnexen** [comment]
I see nothing really wrong left but maybe @Girgias would like to have a look

**Review by adoy — APPROVED** [review]

File: reference/datetime/datetimeinterface/format.xml [hunk 4]
```diff
@@ -174,11 +174,36 @@
             cette année sera utilisée à la place.</entry>
            <entry>Exemples : <literal>1999</literal> ou <literal>2003</literal></entry>
           </row>
+          <row>
+           <entry><literal>X</literal></entry>
+           <entry>
+            Une représentation numérique complète étendue d'une année, d'au moins 4 chiffres,
+            avec <literal>-</literal> pour les années avant l'ère commune
+            et <literal>+</literal> pour les années de l'ère commune.
```
Comment (Girgias) [inline reference/datetime/datetimeinterface/format.xml:182]:
```suggestion
            avec un <literal>-</literal> pour les années avant l'ère commune
            et ubn <literal>+</literal> pour les années de l'ère commune.
```

**Pull request merged**